deploy/olm-catalog/akka-cluster-operator/1.0.2/akka-cluster-operator-certified.v1.0.2.clusterserviceversion.yaml

```diff
@@ -0,0 +1,327 @@
+              resources:
+                - deployments/finalizers
+              verbs:
+                - update
+            - apiGroups:
+                - app.lightbend.com
+              resources:
+                - '*'
+              verbs:
+                - '*'
+      clusterPermissions:
+        - rules:
+            - apiGroups:
+                - ""
+              resources:
+                - pods
+                - services
+                - endpoints
+                - persistentvolumeclaims
+                - events
+                - configmaps
+                - secrets
+                - serviceaccounts
+              verbs:
+                - "*"
+            - apiGroups:
+                - rbac.authorization.k8s.io
+              resources:
+                - roles
+                - rolebindings
+              verbs:
+                - "*"
+            - apiGroups:
+                - ""
+              resources:
+                - namespaces
+              verbs:
+                - get
+            - apiGroups:
+                - apps
+              resources:
+                - deployments
+                - daemonsets
+                - replicasets
+                - statefulsets
+              verbs:
+                - "*"
+            - apiGroups:
+                - monitoring.coreos.com
+              resources:
+                - servicemonitors
+              verbs:
+                - get
+                - create
+            - apiGroups:
+                - apps
+              resourceNames:
+                - akka-cluster-operator
+              resources:
+                - deployments/finalizers
+              verbs:
+                - update
+            - apiGroups:
+                - app.lightbend.com
+              resources:
+                - "*"
+              verbs:
+                - "*"
+          serviceAccountName: akka-cluster-operator
+    strategy: deployment
+  installModes:
+    - supported: true
+      type: OwnNamespace
+    - supported: true
+      type: SingleNamespace
+    - supported: false
+      type: MultiNamespace
+    - supported: true
+      type: AllNamespaces
+  labels:
+    Name: AkkaClusterOperator
+  maturity: stable
+  minKubeVersion: 1.11.0
+  provider:
+    name: 'Lightbend, Inc.'
+  keywords: ["Akka", "Akka Cluster", "Lightbend"]
+  maintainers:
+    - name: 'Lightbend, Inc.'
+      email: info@lightbend.com
+  links:
+    - name: Intro to Akka
+      url: https://doc.akka.io/docs/akka/current/guide/introduction.html
+    - name: Intro to Akka Cluster
+      url: https://doc.akka.io/docs/akka/current/common/cluster.html
+    - name: Akka Cluster demo application
+      url: https://github.com/lightbend/akka-java-cluster-openshift
+    - name: Deploying a Lagom application to OpenShift
+      url: https://developer.lightbend.com/guides/openshift-deployment/lagom/index.html
+  version: 1.0.2
+  replaces: akka-cluster-operator-certified.v1.0.0
```

Comment:
I know the `replaces` thing is kinda wacky, but should that be replacing v1.0.1?  I don't know.  I could never figure this one out...

Reply:
you are right. there is no 1.0.0 in the connect registry, so -certified might want to replace 1.0.1 instead.